frameworks/hdfs/src/main/dist/svc.yml

```diff
@@ -12,21 +11,23 @@ pods:
         cpus: {{JOURNAL_CPUS}}
         memory: {{JOURNAL_MEM}}
         ports:
-          - name: journal_rpc
+          journal-rpc:
             port: {{JOURNAL_NODE_RPC_PORT}}
-          - name: journal_http
+          journal-http:
             port: {{JOURNAL_NODE_HTTP_PORT}}
-        volumes:
-          - path: "journal-data"
-            size: {{JOURNAL_DISK}}
-            type: {{JOURNAL_DISK_TYPE}}
+        volume:
```

Comment:
Yes. There are two versions available:

- singular `volume` (without labels as seen here), shorthand for presumed common case where just one volume is needed
- multiple `volumes` (with labels as in your example) longer form for two or more volumes

The volume labels used in the multi case don't actually do anything for now. Can see both forms used in the exhaustive example.